### DISCUSSION TITLE
playlist + playlist comparison + command line

### DISCUSSION THREAD

**Discussion by balthou1703** [grizzlypeak3d/DJV#500] [Ideas]
it would be nice if we could create a playlist of shots (ex: shot_a_v0, shot_b_v0, shot_c_v0),
Then create a second playlist (ex: shot_a_v1, shot_b_v1, shot_c_v1), and compare them by page up/page down.
It could be even better if we could create them procedurally by command line

**Comment by darbyjohnston** [comment]
Hi, and sorry for the long delay; I just released a new version, 3.0.0, that has some features to support this. You can create playlists/sequences with OpenTimelineIO files and use the new A/B feature to compare them.

**Reply by balthou1703** [reply]
Thank you! can't wait to test it :)
